### PR TITLE
Progress documentation

### PR DESCRIPTION
**Pull request by tomi-msft**
This PR is to update the `README.md` and add a `MIGRATION.md` file to `react-progress`

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |         Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ---------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ButtonProviderAndTheme.fixture.js'>react-components: Button, FluentProvider & webLightTheme</abbr>              | `62.827 kB`<br />`17.599 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `189.299 kB`<br />`52.51 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            | `33.446 kB`<br />`11.033 kB` |
| <samp>react-portal-compat</samp> <br /> <abbr title='bundle-size/PortalCompat.fixture.js'>PortalCompatProvider</abbr>                                                         |   `5.857 kB`<br />`1.978 kB` |
| <samp>react-progress</samp> <br /> <abbr title='bundle-size/Progress.fixture.js'>Progress</abbr>                                                                              |  `11.348 kB`<br />`4.572 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/14e5b74b32528ee2ea7a1992243809de214f5dfc'>14e5b74b32528ee2ea7a1992243809de214f5dfc</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25194/perf-test-react-components/Avatar-mount_base.html">1314</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25194/perf-test-react-components/Avatar-mount.html">1312</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25194/perf-test-react-components/Button-mount_base.html">954</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25194/perf-test-react-components/Button-mount.html">949</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25194/perf-test-react-components/FluentProvider-mount_base.html">1578</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25194/perf-test-react-components/FluentProvider-mount.html">1587</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25194/perf-test-react-components/FluentProviderWithTheme-mount_base.html">633</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25194/perf-test-react-components/FluentProviderWithTheme-mount.html">630</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25194/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">584</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25194/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">582</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25194/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">623</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25194/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">627</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25194/perf-test-react-components/MakeStyles-mount_base.html">1902</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25194/perf-test-react-components/MakeStyles-mount.html">1913</a></td>
            <td>50000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25194/perf-test-react-components/SpinButton-mount_base.html">2510</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/25194/perf-test-react-components/SpinButton-mount.html">2514</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

File: packages/react-components/react-progress/MIGRATION.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+v8 offers a component equivalent to v9's `Progress`. However, the API is slightly different. The main difference is that v9's `Progress` does not have the `label` or `description` props. Instead, the `label` and `description` are located in the `ProgressField` that can be used in conjunction with `Progress`.
+
+Here's how the API of v8's `Progress` compares to the one from v9's `Progress` component:
+
+- `className` => `className`
+- `description` => Use `ProgressField` to use the `description` prop
+- `label` => Use `ProgressField` to use the `label` prop
+- `onRenderProgress` => NOT SUPPORTED
+- `percentComplete` => Use the `value` prop in v9
```
Comment (behowell) [inline packages/react-components/react-progress/MIGRATION.md:13]: (nit) this is all about v9 :)
```suggestion
- `percentComplete` => Use the `value` prop
```

File: packages/react-components/react-progress/MIGRATION.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Progress Migration
+
+## Migration from v8
+
+v8 offers a component equivalent to v9's `Progress`. However, the API is slightly different. The main difference is that v9's `Progress` does not have the `label` or `description` props. Instead, the `label` and `description` are located in the `ProgressField` that can be used in conjunction with `Progress`.
```
Comment (behowell) [inline packages/react-components/react-progress/MIGRATION.md:5]:
```suggestion
v8 offers a component equivalent to v9's `Progress`. However, the API is slightly different. The main difference is that v9's `Progress` does not have the `label` or `description` props. Instead, the `label` and `hint` are located in the `ProgressField` that can be used in conjunction with `Progress`.
```

File: packages/react-components/react-progress/MIGRATION.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+# Progress Migration
```
Comment (spmonahan) [inline packages/react-components/react-progress/MIGRATION.md:1]: IMO a better place for this documentation is in the [v9 doc site](https://github.com/microsoft/fluentui/tree/master/apps/public-docsite-v9/src/Concepts/Migration/FromV8/Components) but this is not a blocker for me.

To me, we should minimize the number of places documentation lives and the doc site is much more visible to our users that a Markdown file.

**Review by spmonahan — APPROVED** [review]

**Pull request merged**